clients/src/main/java/org/apache/kafka/common/network/SslTransportLayer.java

```diff
@@ -423,6 +425,17 @@ private void handshakeFinished() throws IOException {
                 SSLSession session = sslEngine.getSession();
                 log.debug("SSL handshake completed successfully with peerHost '{}' peerPort {} peerPrincipal '{}' cipherSuite '{}'",
                         session.getPeerHost(), session.getPeerPort(), peerPrincipal(), session.getCipherSuite());
+                if (sslInformation == null) {
+                    String cipherSuiteName = session.getCipherSuite();
+                    if ((cipherSuiteName == null) || cipherSuiteName.isEmpty()) {
```

Comment:
removed

Reply:
Are `null` and empty string actually valid values for this method? The documentation doesn't say it is. Same for protocol.

Reply:
It doesn't say they are, but it also doesn't say they aren't. :)

Since the SSL implementation in Java is pluggable, it seems best to make minimal assumptions about what these methods will return.